Vic2ToHoI4/Source/HOI4World/HoI4Localisation.cpp

```diff
@@ -945,15 +952,20 @@ std::string HoI4Localisation::getLanguageCode(const std::string& language)
 void HoI4Localisation::insertScriptedLocalisation(
 	const std::string& localisationKey,
 	const std::string& replacementKey,
-	HoI4::ScriptedLocalisation& scriptedLocalisation
+	HoI4::ScriptedLocalisation& scriptedLocalisation,
+	const std::set<std::string>& majorIdeologies
 ) const
 {
-	const std::regex extractRegex("([A-Z]+)_([a-z]+)_ADJ");
+	const std::regex extractRegex("([A-Z0-9]+)_([a-z]+)_ADJ");
 
 	std::smatch match;
 	std::regex_match(localisationKey, match, extractRegex);
 	const std::string tag = match[1];
 	const std::string ideology = match[2];
+	if (majorIdeologies.count(ideology) == 0)
```

Comment:
count() within if returns bool, no need to compare with zero. 
if (majorIdeologies.count(ideology)){..code..}

Reply:
(to be precise 0 translates to false and non-zero translates to true)